src/modules/reminder/dto/reminder.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsDate, IsEmail, IsNotEmpty, IsNumber, IsString } from 'class-validator';
+
+export class ReminderDto {
+  @ApiProperty({ example: 1 })
+  @IsNumber()
+  @Type(() => Number)
+  id: number;
```

Comment:
```suggestion
  @IsNumber()
  id: number;
```
Для боди это приведение не нужно